doc/classes/LineEdit.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="LineEdit" inherits="Control" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Control that provides single-line string editing.
+		A single-line text editor.
```

Comment:
```suggestion
		A single-line text input.
```
Calling it editor is too exaggerated.

Reply:
I took this from the full desc, so I'll change it there too then.